deploy/runtime/defaults/Generic System_Commercial.txt

```diff
@@ -823,7 +823,7 @@ itc_fed_percent
 1
 1
 1
-26
+0
```

Comment:
Assuming zero because the technology is not defined in Generic?

Reply:
Exactly. In fact the "heat rate" default applies the generic system might be a thermal generator.